payments-core/src/main/java/com/stripe/android/view/CardInputWidget.kt

```diff
@@ -681,6 +681,7 @@ class CardInputWidget @JvmOverloads constructor(
         cardBrandView.tintColorInt = cardNumberEditText.hintTextColors.defaultColor
         var cardHintText: String? = null
         var shouldRequestFocus = true
+        var imeLandscapeFullScreen = true
```

Comment:
does this need to be a member variable if we only access use it in one place?

Reply:
`cardHintText` and `shouldRequestFocus` are same.

Should I write it like this?

```kotlin
if (!getBoolean(
        R.styleable.CardInputView_imeLandscapeFullScreen,
        true
    )
) {
    cardNumberEditText.imeOptions = EditorInfo.IME_FLAG_NO_FULLSCREEN
    expiryDateEditText.imeOptions = EditorInfo.IME_FLAG_NO_FULLSCREEN
    cvcEditText.imeOptions = EditorInfo.IME_FLAG_NO_FULLSCREEN
    postalCodeEditText.imeOptions = EditorInfo.IME_FLAG_NO_FULLSCREEN
}
```